libs/s25main/desktops/dskGameLobby.cpp

```diff
@@ -48,15 +48,55 @@
 namespace {
 enum CtrlIds
 {
-    ID_PLAYER_GROUP_START = 50,
-    ID_SWAP_BUTTON = 80,
-    ID_FIRST_FREE = ID_SWAP_BUTTON + MAX_PLAYERS,
-    ID_GAME_CHAT,
-    ID_LOBBY_CHAT,
-    ID_CHAT_INPUT,
-    ID_CHAT_TAB,
-    TAB_GAMECHAT,
-    TAB_LOBBYCHAT
+    ID_btStartGame,
+    ID_btReturn,
+    ID_chkLockTeams,
+    ID_chkSharedView,
+    ID_chkRandomSpawn,
+    ID_txtAddons,
+    ID_btSettings,
+    ID_txtColPastPlayer,
+    ID_txtColSwap,
+    ID_txtColName,
+    ID_txtColRace,
+    ID_txtColColor,
+    ID_txtColTeam,
+    ID_txtColReady,
+    ID_txtColPing,
+    ID_txtGameName,
+    ID_txtExploration,
+    ID_cbExploration,
+    ID_txtGoods,
+    ID_cbGoods,
+    ID_txtGoals,
+    ID_cbGoals,
+    ID_txtSpeed,
+    ID_cbSpeed,
+    ID_txtNoPreview,
+    ID_txtMapName,
+    ID_miniMap,
+    ID_btPlayerState,
+    ID_btNation,
+    ID_btColor,
+    ID_btTeam,
+    ID_chkReady,
+    ID_txtPing,
+    ID_cbMove,
+    ID_mbLuaLoadError = 100,
```

Comment:
Any reason for this value?

Reply:
Damn, I was sure I removed it - I added it during partial value removal to make sure that they won't collide with anything.